observer/test/observer.test.js

```diff
@@ -184,4 +184,27 @@ describe('observer', () => {
       }])
     })
   })
+
+  describe('observeRetrievalResultStatus', () => {
+    beforeEach(async () => {
+      await pgPools.stats.query('DELETE FROM daily_retrieval_result_status')
+    })
+
+    it('observes retrieval result status', async () => {
+      await observeRetrievalResultStatus(pgPools.stats, {
+        collectRows: async () => [
+          { _time: today(), _field: 'success', _value: 0.5 },
```

Comment:
Can you please use real field names and values, so that the test better documents what we expect on the other side of this integration.

Reply:
f2a7587